content/docs/reference/i3xrocks.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'i3xrocks'
+linkTitle: 'i3xrocks'
+weight: 7
+description: >
+    A small intro into what `i3xrocks` is.
+---
+
+Since i3 is the underlying tiling manager that is used by regolith, the `i3rocks` package can also be used. However since the choice was made to use `Xresources` this package has been forked and `i3xrocks` was born (created by the regolith team).
```

Comment:
correctness/nit: the upstream fork is called `i3blocks`.  https://github.com/vivien/i3blocks

Reply:
I have corrected it and added links to the corresponding repositories